keg/keg.py

```diff
@@ -15,15 +15,70 @@
 # You should have received a copy of the GNU General Public License
 # along with keg. If not, see <http://www.gnu.org/licenses/>
 #
+
+import argparse
 import logging
+import sys
 
 # project
+from keg.exceptions import KegError
+from keg.generator import create_image_description
 from keg.version import __version__
 
 log = logging.getLogger('keg')
 
 
 def main():
-    log.info(
-        'This is keg version {0}'.format(__version__)
+    argparser = argparse.ArgumentParser(
+        description='Create KIWI image description from keg recipe'
+    )
+    argparser.add_argument(
+        '-r', '--recipes-root',
+        dest='recipes_root',
+        help='Root directory of recipes (required)'
```

Comment:
Needs to support multiple, i.e. we either have multiple `--recipes-root` or one with a comma separated list. The help need to be explict about his.